source/cli.ts

```diff
@@ -7,18 +7,54 @@ const cli = meow(`
 	Usage
 	  $ tsd [path]
 
+	  The given directory must contain a package.json and a typings file.
+
+	Info
+	  --help,    -h  Display help text
+	  --version, -v  Display version info
```

Comment:
I prefer to not have short flags for these. `-v` is ambitious to a user and could easily mean `--verbose` too.

Reply:
I had added them originally for consistency and because when I first used `tsd`, I didn't think there was a help prompt, since `tsd -h` had no effect and there was nothing mentioning CLI flags on the readme. I only realized `--help` existed when I was reading the documentation for `meow`. (Which, yes, is on me for not trying `--help`, but still.)

That said, this PR mentions using `tsd --help` for usage information, which would then show the other flags. Are the short flags for `--typings` and `--files` okay?

Reply:
> Are the short flags for --typings and --files okay?

Yes